weblate/trans/models/translation.py

```diff
@@ -1429,6 +1429,14 @@ def add_unit(  # noqa: C901
             self.notify_new(request)
         return result
 
+    def notify_deletion(self, unit, user):
+        change = Change(
+            unit=unit,
```

Comment:
As `Change` object is configured to cascade on unit removal, it will only exist for very short time and will be removed afterwards. In this case, it is probably better to link the event to translation and include the current string and context in `target` and/or `details`.